src/tasks/MultiFormPurgeTask.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace SilverStripe\MultiForm\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\MultiForm\Models\MultiFormSession;
+use SilverStripe\ORM\DataList;
+use SilverStripe\ORM\DataObject;
+
+/**
+ * Task to clean out all {@link MultiFormSession} objects from the database.
+ *
+ * Setup Instructions:
+ * You need to create an automated task for your system (cronjobs on unix)
+ * which triggers the process() method through cli-script.php:
+ * `php framework/cli-script.php MultiFormPurgeTask`
+ * or
+ * `framework/sake MultiFormPurgeTask`
+ *
+ * @package multiform
+ */
+class MultiFormPurgeTask extends BuildTask
+{
+    /**
+     * Days after which sessions expire and
+     * are automatically deleted.
+     *
+     * @var int
+     */
+    private static $session_expiry_days = 7;
```

Comment:
You might want to add `private static $segment = 'MultiFormPurgeTask';` so it won't be namespaced when running the task from command line